Kickstarter-iOS/Views/Cells/LiveStreamContainerMoreMenuIconTextCell.swift

```diff
@@ -0,0 +1,69 @@
+import Library
+import LiveStream
+import Prelude
+
+internal final class LiveStreamContainerMoreMenuIconTextCell: UITableViewCell, ValueCell {
+
+  @IBOutlet private weak var contentStackView: UIStackView!
+  @IBOutlet private weak var iconImageView: UIImageView!
+  @IBOutlet private weak var rootStackView: UIStackView!
+  @IBOutlet private weak var subtitleLabel: UILabel!
+  @IBOutlet private weak var titleLabel: UILabel!
+
+  internal override func awakeFromNib() {
+    super.awakeFromNib()
+
+    self.selectedBackgroundView = UIView()
+      |> UIView.lens.backgroundColor .~ .ksr_navy_700
+  }
+
+  internal func configureWith(value moreMenuItem: LiveStreamContainerMoreMenuItem) {
+    switch moreMenuItem {
+    case .hideChat(let hidden):
+      self.iconImageView.image = Library.image(named: "speech-icon")
+      self.titleLabel.isHidden = true
```

Comment:
Probably something I needed before everything changed. Will fix if we are keeping these cells.